components/prism-php.js

```diff
@@ -33,6 +33,14 @@
 				'punctuation': /\\/
 			}
 		},
+		'class-name-definition': [
+			{
+				pattern: /(\b(?:class|interface|trait)\s+)\b[a-z_]\w*(?!\\)\b/i,
```

Comment:
With `class|interface|trait` covered, you can remove them from the first `class-name` pattern. 

(GH doesn't allow me to make a comment there, so I'm making it here.)

Reply:
Removed